observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,67 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+// note to self: update errata live ID
+* link:https://access.redhat.com/errata/RHSA-2024:0853[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-lokiless-enhancements-1.6"]
+==== Enhanced use of Network Observability Operator without Loki
+You can now do more than ever with the Network Observability Operator using Prometheus metrics and relying less on Loki for storage. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki].
+
+[id="network-observability-custom-metrics-1.6"]
+==== Custom metrics API
+You can create custom metrics out of flowlogs data by using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. For more information, see xref:../../observability/network_observability/metrics-alerts-dashboards.adoc#network-observability-custom-metrics_metrics-dashboards-alerts[Custom metrics] and <FLOW_METRICS_API_REF_LINK_HERE>
+
+[id="network-observability-eBPF-performance-enhancements-1.6"]
+==== eBPF performance enhancements
+With this release, you can experience improved the performances of the eBPF agent, both in terms of CPU and memory. Additionally, the *NetObserv / Health* dashboard integration with eBPF metrics allows you to view alerts in two causes: when the Network Observability eBPF agent hashmap table is full, which cases the eBPF agent processes flows with degraded performance, or when the capacity limiter is triggered. For more information, see xref:../../observability/network_observability/network-observability-operator-monitoring.adoc#network-observability-netobserv-dashboard-ebpf-agent-alerts_network_observability[Using the eBPF agent alert]. 
+
+[id="network-observability-no-show-duplicates-1.6"]
+==== Duplicates no longer shown
```

Comment:
```suggestion
==== Deduper enablement
```

Duplicates is a technical debt; not a feature. We need to emphasize that this is an optimisation !

Reply:
Ok I am going to make the eBPF performance section a bullet list, so I can also add the TCX hooks there as well.